crates/sui-graphql-rpc/src/types/coin.rs

```diff
@@ -7,6 +7,7 @@ use async_graphql::*;
 
 use sui_types::coin::Coin as NativeCoin;
 
+/// The general Coin type.
```

Comment:
Do you intend for this to show up in the graphql schema, or just a rust doc comment? Just asking since it's not showing up in the former

suggestion: /// Some 0x2::coin::Coin<T> Move object

Reply:
Yeah, this needs to move onto the `impl` block!